shimmingtoolbox/cli/image.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import click
+import os
+import nibabel as nib
+
+from shimmingtoolbox.image import concat_data
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+AXES = ['0', '1', '2', '3', '4']
```

Comment:
is "4" really necessary?

Reply:
My thought is if the user has data that is (x,y,z,volumes) and wants to concatenate echoes, then they would need a 5th dimension.

Reply:
ah yes, i forgot that in ST we put the echoes in the 5th dim. OK, i guess it makes sense then